apps/api/v1/lib/utils/isAdmin.ts

```diff
@@ -1,10 +1,39 @@
 import type { NextApiRequest } from "next";
 
 import prisma from "@calcom/prisma";
-import { UserPermissionRole } from "@calcom/prisma/enums";
+import { UserPermissionRole, MembershipRole } from "@calcom/prisma/enums";
+
+export const ScopeOfAdmin = {
+  SystemWide: "SystemWide",
+  OrgOwnerOrAdmin: "OrgOwnerOrAdmin",
+} as const;
```

Comment:
Currently we have decided to go with System-Wide admins and Organization owner or admin, leaving out team admins due to a security concern.